src/main/java/net/openhft/chronicle/wire/YamlWireOut.java

```diff
@@ -85,7 +88,7 @@ public abstract class YamlWireOut<T extends YamlWireOut<T>> extends AbstractWire
     }
 
     protected final YamlValueOut valueOut = createValueOut();
-    protected final StringBuilder sb = new StringBuilder();
+    protected final ScopedResourcePool<StringBuilder> sb = StringBuilderPool.createThreadLocal(1);
```

Comment:
Note that this probably doesn't need to be a thread-local given that YamlWireOut is single-threaded anyway. Making a single-threaded implementation of ScopedResourcePool is trivial.